repos/rust-lang/calendar.toml

```diff
@@ -0,0 +1,20 @@
+org = "rust-lang"
+name = "calendar"
+description = "Calendars for Rust project teams"
+bots = ["rustbot"]
+
+[access.teams]
+community = "maintain"
+compiler = "maintain"
+compiler-contributors = "maintain"
+crates-io = "maintain"
+docs-rs = "maintain"
+infra = "maintain"
+lang = "maintain"
+leadership-council = "maintain"
+libs = "maintain"
+release = "maintain"
+triagebot = "maintain"
```

Comment:
This shouldn't be necessary with `bots = ["rustbot"]`

Reply:
Fixed - had just copied the list from the forge repo.